field/src/packed.rs

```diff
@@ -96,6 +96,11 @@ where
     fn doubles(&self) -> Self {
         *self * Self::Scalar::TWO
     }
+
+    #[inline]
+    fn is_zero(&self) -> bool {
```

Comment:
In line with `Field::ZERO` vs `PackedField::ZEROS`:
```suggestion
    fn is_zeros(&self) -> bool {
```